static/css/components/buttonSearch.less

```diff
@@ -0,0 +1,51 @@
+/**
+ * Search button and text input
+ * [TODO: Link to documentation]
+ */
+
+.search-div {
+  // @grey-fafafa;
+  background-color: @white;
+  border: 1px solid @dark-beige;
+  border-radius: .3em;
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding: .2%;
+  padding-left: .5%;
+  width: 50%;
+}
+
+.search-btn-icon {
+  background: url(/static/images/search-icon.svg) center no-repeat;
+  width: 35px;
+  height: 45px;
+  cursor: pointer;
+  border: 0;
+  flex: 0 0 auto;
+}
+
+.search-input {
+  margin: 0 !important;
+  border: 0;
+  outline: none;
+  background: transparent;
+  color: @grey !important;
+  flex-grow: 1;
+  font-size: 1em !important;
+  height: 45px;
+  max-width: 93% !important;
+}
+
+.search-btn {
```

Comment:
This doesn't really make sense in this file ; this css file is just about the `searchbox`; this is kind of just a random other button :P This should be based on our exisitng cta-btn, which correctly handles things like font size, hover states etc.

See where `.cta-btn` is defined, and add a new one, `cta-btn--search`. Then to the element, add `class="cta-btn cta-btn--search"`